src/solc_0.5/Sand.sol

```diff
@@ -4,7 +4,7 @@ import "./Sand/erc20/ERC20ExecuteExtension.sol";
 import "./Sand/erc20/ERC20BaseToken.sol";
 import "./Sand/erc20/ERC20BasicApproveExtension.sol";
 
-contract Sand is ERC20ExecuteExtension, ERC20BasicApproveExtension, ERC20BaseToken {
+contract Sand is ERC20BasicApproveExtension,ERC20ExecuteExtension, ERC20BaseToken {
```

Comment:
Hey @capedcrusader21 I would prefer not to change older contracts that are not relevant to the PR... Is this unavoidable?

Reply:
There was a compilation issue, This resolved it.